.chloggen/add_windows_container_images.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: packaging
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Start producing images for Windows 2019 and 2022 on amd64
```

Comment:
```suggestion
note: Start producing Docker images for Windows 2019 and 2022 on amd64
```

Reply:
:)
```suggestion
note: Start producing container images for Windows 2019 and 2022 on amd64
```

Reply:
Going with @mowies suggestion to be more neutral.